Tools/process_run.py

```
@@ -0,0 +1,135 @@
+        interpolation="bilinear",
+        origin="lower",
+        extent=[xedges[0], xedges[-1], yedges[0], yedges[-1]],
+    )
+    plt.colorbar(im, ax=ax)
+    ax.set_xlabel("$x$ [m]")
+    ax.set_ylabel("$z$ [m]")
+    ax.set_title(metric.name)
+    ax.grid(alpha=0.25)
+
+    # Plot a scatter plot of the metric values.
+    fig, ax = plt.subplots(figsize=(16, 8))
```

Ruff (0.14.8)

Unpacked variable `fig` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)